### PR TITLE
Extend1679

### PR DESCRIPTION
**Pull request by mirraaj**
Returns the log probability of one observation (x) for one of the Gamma's dimensions.

### CODE REVIEW

**Comment by mirraaj** [comment]
@rcurtin @zoq This is an extension of #1679 . Do we need a test for this as well?

**Comment by rcurtin** [comment]
If you added new functionality, then yes.

**Comment by mirraaj** [comment]
@rcurtin If we merge #1679, this PR becomes just an extension of introducing function in Gamma Distribution. Let me know if you require justification/rectification if needed. Thanks

**Comment by rcurtin** [comment]
Right, and if you added new functions to `GammaDistribution` then they should also be tested. :+1:

**Comment by mirraaj** [comment]
@ShikharJ, @rcurtin, @zoq This is a small PR to extend #1679. I have also added the test for this addition. Please let me know your views about this. Thanks

File: src/mlpack/core/dists/gamma_distribution.hpp [hunk 1]
```diff
@@ -171,6 +172,14 @@ class GammaDistribution
      */
     void LogProbability(const arma::mat& observations,
                         arma::vec& LogProbabilities) const;
+    /*
```
Comment (ShikharJ) [inline src/mlpack/core/dists/gamma_distribution.hpp]: Please include a newline here. Also another `*` I think.

Comment (mirraaj) [reply src/mlpack/core/dists/gamma_distribution.hpp]: @ShikharJ I have included the changes.

**Comment by mirraaj** [comment]
@rcurtin @zoq Any comments?

**Comment by rcurtin** [comment]
The only distributions left for this support now are `LaplaceDistribution` and `RegressionDistribution`.  In both cases the changes should be really easy, just like this PR.

**Comment by mirraaj** [comment]
Yeah, I am working on it. I will push a PR soon. Thanks

**Pull request merged**